src/com/robibp/trening/EvidencijaDrive.java

```diff
@@ -1,29 +1,28 @@
 package com.robibp.trening;
 
+import java.util.ArrayList;
 import java.util.Scanner;
 
 class EvidencijaDrive {
 
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         MenuChooser menuChooser = new MenuChooser();
-        Klinac[] imena = new Klinac[100];
+        MenuEnum ponavljanje = null;
```

Comment:
Zasto ovo zoves `ponavljanje`. Prvenstvena namjera te varijable je da ti kaze sto je korisnik odabrao a to sto ju ti koristis za nekakvo ponavljanje je popratna pojava. Nazovi ju `odabraniMenu`